src/components/Screen.js

```diff
@@ -2,22 +2,16 @@
 /* eslint-disable react/prop-types */
 import React from 'react';
```

Comment:
- [ ] Kindly do not disable `eslint` rules as they are there to improve your code quality. :pray: